src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -957,21 +959,20 @@ public static EntityPlayer getCraftingPlayer()
     {
         return craftingPlayer.get();
     }
-    public static ItemStack getContainerItem(ItemStack stack)
+    @Nonnull
+    public static ItemStack getContainerItem(@Nonnull ItemStack stack)
     {
-        if (stack == null) return null;
-
         if (stack.getItem().hasContainerItem(stack))
```

Comment:
Is AIR the only invalid itemStack? (Sorry I don't have a 1.11 workspace to actually look at itemStack, hotel wifi is rubbish and my data plan is used up). My understanding (based on early rumors) was that any itemstack could become invalid e.g. by getting down to stacksize 0.  Sorry again to bother you if that's wrong.